src/Components/Cluster/Cluster.spec.ct.js

```diff
@@ -63,7 +64,7 @@ describe('cluster page', () => {
     // renders table component
     cy.get(RULES_TABLE).should('have.length', 1);
     // test how many rows were rendered
-    // TODO use checkRowCounts (currently not working, probably related to federated modules)
+    checkRowCounts(singleClusterPageReport.report.data.length);
     cy.getAllRows().should(
```

Comment:
If we already use `checkRowCounts` we can get rid of `cy.getAllRows().should(...`